### PR TITLE
Documenting the installation of the provider itself

### PR DESCRIPTION
**Pull request by dcoraboeuf**
### Description of your changes

Documenting how to install the provider itself in Crossplane.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

Documentation only.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -57,7 +57,16 @@ helm repo update
...
 ```
 
-## Install the DigitalOcean Crossplane Provider 
+## Install the DigitalOcean Crossplane Provider
+
+1. Find the provider installation file at [provider-digitalocean/examples/provider/install.yaml](./examples/provider/install.yaml)
+2. Run the installation:
+
+```console
+kubectl apply -f provider-digitalocean/examples/provider/install.yaml
```
Comment (ADustyOldMuffin) [inline INSTALL.md:66]: Can we update this to be the url to the raw Github content? That way you don't have to download the file.

Comment (dcoraboeuf) [reply INSTALL.md]: I've adapted the links (mine and the config one, which was already there).

As for the `install.yaml` file, it's part of this PR.

File: INSTALL.md [hunk 2]
```diff
@@ -57,13 +57,22 @@ helm repo update
...
+```
+
+## Configure the DigitalOcean Crossplane Provider 
+1. Find the provider config file at [examples/provider/config.yaml](./examples/provider/config.yaml)
 1. [Create a DigitalOcean personal access token](https://docs.digitalocean.com/reference/api/create-personal-access-token/)
 1. Encode that token using base64, and in the `config.yaml` file, replace `BASE64ENCODED_PROVIDER_CREDS` with your encoded token
 1. Create a new Secret and ProviderConfig with 
 ```console
-kubectl apply -f provider-digitalocean/examples/provider/config.yaml
+kubectl apply -f https://raw.githubusercontent.com/crossplane-contrib/provider-digitalocean/main/examples/provider/config.yaml
```
Comment (ADustyOldMuffin) [inline INSTALL.md:75]:
```suggestion
kubectl apply -f config.yaml
```
This one users need to actually replace values so we can remove the Github content.

**Comment by ADustyOldMuffin** [comment]
Sorry for the lateness! You can follow the DCO link in the checks to make sure you pass that one.

**Review by kimschles — APPROVED** [review]
Thank you for this contribution, @dcoraboeuf. It's very helpful.

**Pull request merged**